app/backend/src/couchers/models.py

```diff
@@ -867,26 +875,35 @@ def __repr__(self):
         return f"Message(id={self.id}, time={self.time}, text={self.text}, author={self.author}, conversation={self.conversation})"
 
 
-class Complaint(Base):
+class Report(Base):
```

Comment:
I think `Complaint` was clearer, since the noun `Report` means more like a monthly summary to me.

Reply:
I could make it `ContentReport`?